app/templates/packages/view.html

```diff
@@ -248,6 +250,37 @@ <h1 class="display-3">
 				{% endif %}
 
 				{% if screenshots or package.check_perm(current_user, "ADD_SCREENSHOTS") or package.video_url %}
+					{% if screenshots or package.check_perm(current_user, "ADD_SCREENSHOTS") %}
+						<div class="modal fade" id="galleryModal" tabindex="-1" role="dialog" aria-hidden="true">
+							<div class="modal-dialog modal-lg">
```

Comment:
I suggest xl rather than lg:

```suggestion
							<div class="modal-dialog modal-xl">
```